data/Language/antlr4.yaml

```diff
@@ -0,0 +1,46 @@
+identifier: antlr4
+creation_date: 2017
+wikidata: Q295801
+full_name: Another Tool For Language Recognition
+grammar_file: https://github.com/antlr/grammars-v4/tree/master/antlr4
+aliases:
+  - antlr4
+extensions:
+  - g4
+line_continuation:
+  - backslash
+  - vertbar
+delimiters:
+  # comment delimiters:
+  - double_slash
+  - multiline_slash_star
+  - multiline_double_slash_star
+  # string delimiters:
+  - single_quote_slash_escape
+  # block delimiters:
+  - curly_braces
+  - round_braces
+  - square_braces
+  - antlr_colon_semicolon
+keywords:
+  - import
+  - fragment
+  - lexer
+  - parser
+  - grammar
+  - returns
+  - locals
+  - throws
+  - catch
+  - finally
+  - mode
+  - options
+  - tokens
+  - skip
+  - more
+  - popMode
+  # x is a parameter
```

Comment:
I don't think we have to mention this, do we?
It could be re-written to `mode()`, `pushMode()`, etc, no?

Reply:
The idea is to show that it doesn't take any default parameter.Check [this](https://www.antlr.org/api/Java/org/antlr/v4/runtime/Lexer.html#pushMode(int))